### DISCUSSION TITLE
How to authenticate under service account?

### DISCUSSION THREAD

**Discussion by vitalyk-multinarity** [watermint/toolbox#701] [Q&A] [answered]
I plan to use tbx utility to edit Google Sheets in automatic CI flow.
I prefer to use a GCP service account instead of my company's Gmail account.
If/how possible to do that?

Thanks,
Vitaly

**Comment by watermint** [comment, marked as answer]
Hi. At least my knowledge, it's not possible.

I'm not a specialist of Google Workspace or GCP license.
But in my understanding, Google Sheet is the part of Workspace product not part of GCP.
If you want isolate process from your Workspace account, you should prepare Workspace service account.
